### PR TITLE
fix Struct.toFields()

### PR DESCRIPTION
**Pull request by Comdex**
o1js companion of https://github.com/o1-labs/o1js-bindings/pull/150

### CODE REVIEW

**Comment by Comdex** [comment]
Updated hard-coded keys for testing. Having `Struct.toFields()` output in the order defined by the properties in the js object allows us to more straightforwardly and intuitively understand the placement of our defined variables within the 8 storage slots in the contract, which is meaningful. We can easily set some preconditions on the state of third-party contracts without relying on their code (which is exactly how our team operates), and we can also directly review whether the contract's state meets the requirements. If not done this way, you might frequently need to spend some time understanding the meanings of the 8 on-chain fields.

**Comment by mitschabaude** [comment]
Totally agree @Comdex! I only want to change the order in the example Structs, by changing how they are defined, so we get rid of the verification key changes. I can do it if you invite me to your repo :D

**Comment by Comdex** [comment]
@mitschabaude Invited!😃

**Review by mitschabaude — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,8 +13,18 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
+
+
  -->
 
+## [Unreleased](https://github.com/o1-labs/o1js/compare/c2f392fe5...HEAD)
+
+### Breaking changes
+
+- Changes to some verification keys caused by changing the way `Struct` orders object properties. https://github.com/o1-labs/o1js/pull/1124 [@Comdex](https://github.com/Comdex)
+  - Existing verification keys and behaviour can be recovered by changing the order of properties in your Struct definitions to be alphabetical
```
Comment (barriebyron) [inline CHANGELOG.md:25]:
```suggestion
  - You can recover existing verification keys and behavior by changing the order of properties in your Struct definitions to be alphabetical
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,8 +13,18 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
+  - Also removes the `customObjectKeys` option from `Struct`
```
Comment (barriebyron) [inline CHANGELOG.md:26]:
```suggestion
  - The `customObjectKeys` option is removed from `Struct`
```

File: CHANGELOG.md (hunk 1 above)
Comment (barriebyron) [reply CHANGELOG.md:25]: @mitschabaude, thanks for the changelog updates.

**Review by ymekuria — APPROVED** [review]

**Pull request merged**